service/logger/audit/constants.go

```diff
@@ -52,6 +53,7 @@ const (
 	ActionTypeUpdate
 	ActionTypeDelete
 	ActionTypeRewrap
+	ActionTypeSoftDelete
```

Comment:
We've considered [deactivations historically to be updates for audit purposes](https://github.com/opentdf/platform/blob/9e7d5b972f9e55752b79e3191b7625b7389b1b2d/service/policy/attributes/attributes.go#L165). Could you please align all deactivation audit calls to this new soft delete or update the key soft deletes to use the existing `ActionTypeUpdate`?